src/applications/vaos/components/PastAppointmentsDateDropdown.jsx

```diff
@@ -33,9 +33,8 @@ export default function PastAppointmentsDateDropdown({
         type="button"
         className="vads-u-display--inline-block vads-u-margin-left--2"
         aria-label="Update my appointments list"
-        disabled={currentRange === dateRangeIndex}
         onClick={() => {
-          onChange(dateRangeIndex);
+          currentRange !== dateRangeIndex ? onChange(dateRangeIndex) : false;
```

Comment:
I think the linter is complaining because you're not using the result of your [ternary](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) (which would either be the output of `onChange()` or `false`). Ternaries are generally meant to be used to set a variable or choose a value to be used in a conditional. This would make the warning go away, I think:

```js
onClick={() => currentRange !== dateRangeIndex ? onChange(dateRangeIndex) : false}
```
Removing the braces means that you're returning the result of the ternary. It's still not really used in our code, but the linter can't tell. But I would switch it to this, because I think it's clearer about what you're trying to do:

```js
onClick={() => {
  if (currentRange !== dateRangeIndex) {
    onChange(dateRangeIndex)
  }
}}
```

We do use ternaries throughout the code, and they can at times make things more concise, but there's no problem with using plain `if` statements. They're easier to read in lots of cases.

Reply:
Thank @jbalboni, I'll switch it over to the regular if statement.